Tests/APITesters/CustomEntitlementComputationSwiftAPITester/SwiftAPITester/AttributionAPI.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  AttributionAPI.swift
+//  SwiftAPITester
+//
+//  Created by Joshua Liebowitz on 6/13/22.
+//
+
+import Foundation
+import RevenueCat_CustomEntitlementComputation
+
+var attribution: Attribution!
+
+func checkAttributionAPI() {
+    attribution.setAttributes([String: String]())
```

Comment:
I guess it won't work because these are instance methods and there's no public initializer for the type, right?

Reply:
Oooh I missed this. Yeah `Purchases.attribution` is not exposed but I'll hide this too.
Good catch!

Reply:
Looking at this now. Hiding `Attribution` altogether would require a lot of code changes.
It's impossible to use this because you can't access `Purchases.attribution`, so I think we're good. 

I'll remove this file from `APITester` because we don't actually care about this type in here.